lib/active_job/queue_adapters/amazon_sqs_adapter.rb

```diff
@@ -11,16 +11,16 @@ def enqueue(job)
         _enqueue(job)
       end
 
-      def enqueue_at(job, timestamp, opts={})
+      def enqueue_at(job, timestamp)
         delay = (timestamp - Time.now.to_f).floor
         raise ArgumentError, 'Unable to queue a job with a delay great than 15 minutes' if delay > 15.minutes
-        _enqueue(job, delay_seconds: delay)
+        _enqueue(job, {}, delay_seconds: delay)
```

Comment:
I dont think this is correct - should probably be nil (or job.serialize)

Reply:
Duplicate comment, but fixed.